services/src/main/scala/cromwell/pipeline/service/VersioningException.scala

```diff
@@ -1,13 +1,16 @@
 package cromwell.pipeline.service
 
-sealed abstract class VersioningException extends Exception {val message: String}
+sealed abstract class VersioningException extends Exception {
+  val message: String
+  override def getMessage() = message
+}
```

Comment:
There is another way to make it
```suggestion
sealed abstract class VersioningException(message: String) extends Exception(message)
```
However, it will force you to write
`case class ProjectException(message: String) extends VersioningException(message)`
instead of
`case class ProjectException(message: String) extends VersioningException()`
However, in my opinion, it is a little bit better.

Reply:
Agree, it looks like better